package.json

```diff
@@ -1,69 +1,7 @@
 {
-  "name": "stoat-action",
-  "version": "0.0.4",
-  "private": false,
-  "description": "Supercharge your CI / CD pipelines",
-  "main": "lib/app.js",
-  "scripts": {
-    "build": "npm run gen-types && tsc",
-    "format": "prettier --write '**/*.{ts,json}'",
-    "format-check": "prettier --check '**/*.{ts,json}'",
-    "lint": "eslint src/**/*.ts",
-    "package": "npm run gen-types && ncc build --source-map --license licenses.txt",
-    "gen-types": "json2ts -i 'src/schemas/**/*.json' -o src/schemas",
-    "test": "jest --coverage",
-    "all": "npm run build && npm run format && npm run lint && npm run package && npm test",
-    "prepare": "husky install"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/stoat-dev/stoat-action.git"
-  },
-  "keywords": [
-    "github",
-    "actions",
-    "ci",
-    "cd"
-  ],
-  "author": "Stoat Dev",
-  "license": "MIT",
-  "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/exec": "^1.1.1",
-    "@actions/github": "^5.1.1",
-    "ajv": "^8.11.2",
-    "bluebird": "^3.7.2",
-    "cross-fetch": "^3.1.5",
-    "form-data": "^4.0.0",
-    "handlebars": "^4.7.7",
-    "mime-types": "^2.1.35"
-  },
-  "devDependencies": {
-    "@octokit/openapi-types": "^14.0.0",
-    "@trivago/prettier-plugin-sort-imports": "^3.4.0",
-    "@types/bluebird": "^3.5.37",
-    "@types/js-yaml": "^4.0.5",
-    "@types/mime-types": "^2.1.1",
-    "@types/node": "^18.11.9",
-    "@vercel/ncc": "^0.34.0",
-    "eslint": "^8.28.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-github": "^4.4.1",
-    "eslint-plugin-jest": "^27.1.5",
-    "husky": "^8.0.2",
-    "jest": "^29.3.1",
-    "js-yaml": "^4.1.0",
-    "json-schema-to-typescript": "^11.0.2",
-    "lint-staged": "^13.0.3",
-    "prettier": "2.7.1",
-    "ts-jest": "^29.0.3",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.4.4",
-    "utility-types": "^3.10.0"
-  },
-  "lint-staged": {
-    "*.{ts,js,json}": [
-      "prettier --write"
-    ]
-  }
+  "private": true,
+  "workspaces": [
+    "action",
+    "cli"
```

Comment:
Should docs be listed as a workspace as well? Totally fine if that's meant for a separate PR.

Reply:
I guess I can see a static dependency introduced? Added it.